src/shared/components/enhancedReactTable/EnhancedReactTable.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import {Table as DataTable} from "reactableMSK";
```

Comment:
If this feature will ultimately need to scale to handle large data sets, we should use FBs FixedDataTable, unless we're ok with pagination.